File: Back/src/ProEventos.Domain/Entities/Palestrante.cs
using System.Collections.Generic;

namespace ProEventos.Domain.Entities
{
    public class Palestrante :BaseEntity
    {
        public string Nome { get; set; }
        public string MiniCurriculo { get; set; }
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public IEnumerable<RedeSocial> RedeSociais { get; set; }
         public IEnumerable<Palestrante_Evento> PalestrantesEventos { get; set; }
    }
}

File: ProEventos-em-Vue/src/models/Palestrante.ts
import { Palestrante_Evento } from "./Palestrante_Evento"
import { RedeSocial } from "./RedeSocial"

export interface Palestrante {
    id: number
    nome: string
    miniCurriculo: string
    imagemURL: string
    telefone: string
    email: string
    redeSociais: Array<RedeSocial>
    palestrantesEventos: Array<Palestrante_Evento>
}